fearless/Common/Storage/Migration/NetworkConnectionsMigrator.swift

```diff
@@ -0,0 +1,16 @@
+import SoraKeystore
+
+final class NetworkConnectionsMigrator: Migrating {
+    private(set) var settings: SettingsManagerProtocol
+
+    init(settings: SettingsManagerProtocol) {
+        self.settings = settings
+    }
+
+    func migrate() throws {
+        let selectedConnection = settings.selectedConnection
+        if ConnectionItem.deprecatedConnections.contains(selectedConnection) {
```

Comment:
This doesn't take into account connection type. We should get type from selected connection and find first connection with the same type which is available. Let's also add test for this case ;)